company-kata/src/test/java/org/eclipse/collections/companykata/Exercise4Test.java

```diff
@@ -95,7 +97,9 @@ public void filterOrderValues()
         List<Order> orders = this.company.getMostRecentCustomer().getOrders();
```

Comment:
Was this deliberately not updated to "var" 
var orders = this.company.getMostRecentCustomer().getOrders()

Reply:
Yes, I left this call typed so the user has the specific type information available so they known which APIs they have to use in their code.